Gemfile.lock

```diff
@@ -109,6 +126,7 @@ DEPENDENCIES
   json-schema
   kramdown (~> 2.1)
   less (= 2.4.0)
+  rubocop (>= 0.83.0)
```

Comment:
It appears `rubocop` is in the Gemfile twice - `>= 0.83.0` here, and `0.85.0` above.

Reply:
This is the Gemfile.lock file, and not directly changed by me, but is a result of the `bundle install` command being issued.  (I agree, it is odd).